### DISCUSSION TITLE
HLS to File / Path

### DISCUSSION THREAD

**Discussion by winkmichael** [bluenviron/mediamtx#862] [Questions]
Currently HLS is served via the default port 8888, it would be nice to have the M3U8/TS files written to a path.

Currently we proxy, but just being able to write to disk could be handy.

Thanks for this cool project!

**Comment by aler9** [comment]
Hello the reason why the M3U8 / TS files are not written to disk is that RAM is way faster than HDD / SDD when reading and writing, and in this way the server is much faster than any other HLS server.

To save these files to disk, you can use a reverse proxy (such as Squid) between the browser and the server, and configure it in order to save served files to a path.

**Comment by winkmichael** [comment]
Thanks for your thoughtful comment. Your answer makes complete sense, with NGINX we use a RAM disk to address the latency / performance aspect.

[a comment by an automated account is not shown]
